wic/aos-vm.wks.in

```diff
@@ -2,7 +2,7 @@
 # long-description: Creates a partitioned EFI disk image for genericx86* machines
 part --source aos-bootimg-efi --sourceparams="loader=grub-efi,initrd=core-image-tiny-initramfs-genericx86-64.cpio.gz,version=${BOOT_IMAGE_VERSION}" --ondisk hda --label msdos --active --align 1024 --size 128M
 part --source aos-bootimg-efi --sourceparams="loader=grub-efi,initrd=core-image-tiny-initramfs-genericx86-64.cpio.gz,version=${BOOT_IMAGE_VERSION}" --ondisk hda --label msdos --align 1024 --size 128M
-part / --source rootfs --ondisk hda --fstype=ext4 --label platform --align 1024 --size 1024M --exclude-path ./var/ ./home/
+part / --source rootfs --ondisk hda --fstype=ext4 --label platform --align 1024 --size 1024M
```

Comment:
may be could have two wks files for image with SElinux and w/o

Reply:
Thank you for the close review. Good solution, I've added 2 separate files.

Reply:
> What is the problem with labeling with remove var and home folders?

That because in case of using --exclude directive, script (poky/scripts/lib/wic/plugins/source/rootfs.py) makes a copy of different parts from rootfs, to exclude directories and at the same time ignores SELinux labeling, that performed on image_preprocess stage.